README.md

```diff
@@ -1,2 +1,82 @@
 # amarillo-gtfs-generator
-Generate GTFS from carpools as standalone (Docker) service
+Generate GTFS from carpools as standalone (Docker) service.
```

Comment:
Say
- this is a non-public backend service called from the Amarillo FastAPI application
- it is part of docker compose, that's the easierst, no need to go into these details

Reply:
Include a diagram